### PR TITLE
CI bug for main/release

### PR DESCRIPTION
**Pull request by cbrit**

### CODE REVIEW

File: .github/workflows/integration_tests.yml [hunk 1]
```diff
@@ -95,14 +94,26 @@ jobs:
...
+          images: ${{ env.IMAGE_NAME }}
       - name: Make upload dir
         run: mkdir -p ${{ env.UPLOAD_DIR }}
       - name: build-and-push
         uses: docker/build-push-action@v2
         with:
           file: Dockerfile
-          push: false
-          tags: steward:prebuilt
+          push: ${{ github.event_name != 'pull_request' }}
```
Comment (EricBolten) [inline .github/workflows/integration_tests.yml:101]: I suppose my main question here is: do we need to push to Docker at all for steward and hardhat images in our CI workflows? Couldn't we just build the steward and hardhat docker images as artifacts and remove the push aspect entirely?

**Review by EricBolten — COMMENTED** [review]
A question around Docker usage.

File: .github/workflows/integration_tests.yml (hunk 1 above)
Comment (cbrit) [reply .github/workflows/integration_tests.yml:101]: my thinking was we wanted the `steward:main` tag to get pushed when merging to main?

afa hardhat, maybe not so much

Comment (EricBolten) [reply .github/workflows/integration_tests.yml:101]: We don't really need to publish any docker images unless something else needs to pull them. Steward as a repo is effectively a leaf node in the dependency tree.

**Review by EricBolten — APPROVED** [review]
LGTM

**Pull request merged**